mlir/tools/mlir-miopen-driver/mlir-miopen-lib.cpp

```diff
@@ -47,8 +58,8 @@ static void strToTokens(const std::string &arguments,
 typedef void *MlirHandle;
 
 extern "C" MlirHandle CreateMlirHandle(const char *arguments) {
-  registerDialect<miopen::MIOpenDialect>();
-  registerDialect<StandardOpsDialect>();
+  mlir::registerAllDialects();
+  mlir::registerAllPasses();
```

Comment:
@jerryyin I guess this is changed because `MlirLowerBin` now basically requires all dialects and pass to operate?

Reply:
That's right. Or I can make two API functions one for cpp and another for bin. I used one function instead of two under the consideration that the cpp backend (does not go with active development, therefore) will not persist a long time.

Reply:
it's fine to do all the needed initialization here.